### DISCUSSION TITLE
詢問明天就要交的python作業二

### DISCUSSION THREAD

**Discussion by Rebecca111419** [1101-GS4719-C/Python-Community-GS4719-C#1701] [Q&A]
想請問 我return output 後面是這樣寫，顯示錯誤，但我看不出我錯在哪邊，不知如何debug....

rows_1 = 5
print(f"rows = {rows_1}")
print(hw2_0(rows_1))

rows_2 = 6
print(f"\nrows = {rows_2}")
print(hw2_0(rows_2))

rows_3 = 9
print(f"\nrows = {rows_3}")
print(hw2_0(rows_3))

**Comment by Rebecca111419** [comment]
另外，若是在colab有跑出正確結果，但是在github尚不符合 沒通過，這樣是可以的嗎

**Reply by ATREE01** [reply]
如果在github上顯示不符合的話那就相當於是沒有通過，同常程式類型的作業如果是使用自動批改的話只要沒有通過都是沒有分數的，除非老師有額外手動批改。
